### PR TITLE
Update monitor plugin to use lisk-framework - Closes #6131

### LINKED ISSUE #6131: Update monitor plugin to use lisk-framework
### Description

Update monitor plugin tests to use created testing utilities.

### Motivation

- All testing utilities should be used in the framework to test the usability of the functions

### Acceptance Criteria

- Plugin functional tests are using the utility functions created

### PR DESCRIPTION
**Pull request by ishantiw**
### What was the problem?

This PR resolves #6131

### How was it solved?

♻️ Use testing in monitor plugin from framework #6131 9b10acc

### How was it tested?

`npm run test`

### CODE REVIEW

**Review by nazarhussain — APPROVED** [review]

File: framework-plugins/lisk-framework-monitor-plugin/test/unit/forks.spec.ts [hunk 1]
```diff
@@ -12,33 +12,22 @@
...
-		options: {},
-	} as any;
+	const {
+		mocks: { channelMock },
+	} = testing;
 
 	beforeEach(async () => {
 		monitorPluginInstance = new (MonitorPlugin as any)();
-		await monitorPluginInstance.load(channelMock);
+		await monitorPluginInstance.load(channelMock as any);
```
Comment (shuse2) [inline framework-plugins/lisk-framework-monitor-plugin/test/unit/forks.spec.ts:30]: why do we need `any` here?

File: framework/test/utils/node/state_store_mock.ts [hunk 2]
```diff
@@ -71,9 +71,9 @@ export class StateStoreMock {
 				const index = this.accountData.findIndex(acc => acc.address.equals(address));
 				if (index > -1) {
 					this.accountData[index] = account;
-					return;
```
Comment (shuse2) [inline framework/test/utils/node/state_store_mock.ts:74]: i think this `return` was correct?

File: framework/test/utils/node/state_store_mock.ts [hunk 3]
```diff
@@ -85,6 +85,7 @@ export class StateStoreMock {
 			// eslint-disable-next-line @typescript-eslint/require-await
 			set: async (key: string, value: Buffer): Promise<void> => {
 				this.chainData[key] = value;
+				return Promise.resolve();
```
Comment (shuse2) [inline framework/test/utils/node/state_store_mock.ts:88]: if it's `async` there is no need to explicitly return resolve?

File: framework/test/utils/node/state_store_mock.ts [hunk 4]
```diff
@@ -71,9 +71,9 @@ export class StateStoreMock {
 				const index = this.accountData.findIndex(acc => acc.address.equals(address));
 				if (index > -1) {
 					this.accountData[index] = account;
-					return;
 				}
 				this.accountData.push(account);
+				return Promise.resolve();
```
Comment (shuse2) [inline framework/test/utils/node/state_store_mock.ts:76]: same as below

File: framework-plugins/lisk-framework-monitor-plugin/test/unit/forks.spec.ts (hunk 1 above)
Comment (ishantiw) [reply framework-plugins/lisk-framework-monitor-plugin/test/unit/forks.spec.ts:30]: channelMock is just an object and TS expects and throws `is not a class derived from 'BaseChannel'`. So either we can create a channel mock of a derived from BaseChannel or typecast to any

Comment (ishantiw) [reply framework-plugins/lisk-framework-monitor-plugin/test/unit/forks.spec.ts:30]: channelMock is just an object and TS expects and throws `is not a class derived from 'BaseChannel'`. So either we can create a channel mock of a derived from BaseChannel or typecast to any

File: framework/test/utils/node/state_store_mock.ts (hunk 2 above)
Comment (ishantiw) [reply framework/test/utils/node/state_store_mock.ts:74]: It was throwing error for reason earlier after rebasing but now its not, i'll keep it to return

**Review by shuse2 — APPROVED** [review]

**Pull request merged**